omnisend-for-ninja-forms/includes/Client/class-omnisendapiclient.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Omnisend Api client
+ *
+ * @package OmnisendNinjaFormsPlugin
+ */
+
+declare(strict_types=1);
+
+namespace Omnisend\NinjaFormsAddon\Client;
+
+/**
+ * Omnisend Api client.
+ */
+class OmnisendApiClient {
+
+	/**
+	 * Creates an Omnisend contact.
+	 *
+	 * @param array $body The contact data.
+	 * @return mixed The response from the API.
+	 */
+	public function create_omnisend_contact( array $body ) {
+		$api_key  = get_option( 'omnisend_api_key', null );
```

Comment:
I would suggest checking whether API key is actually present. If not, log an error.